web/apps/labelstudio/src/pages/Settings/AnnotationSettings.js

```diff
@@ -57,23 +57,13 @@ export const AnnotationSettings = () => {
 
           <Form.Row columnCount={1} style={{ borderTop: '1px solid #f1f1f1' }}>
             <br />
-            <Elem name={'header'}>Live Predictions</Elem>
+            <Elem name={'header'}>Prelabeling</Elem>
             <div>
               <Toggle
-                label="Use predictions to pre-label tasks"
+                label="Use predictions to prelabel tasks"
                 description={
                   <span>
-                    Enable and select which set of predictions to use for prelabeling.
-                    Predictions will be pre-loaded in {" "}
```

Comment:
Looks like a helpful info given the diversity of affected workflows (and I believe some were missing here anyway) + links to documentation.
Can we maybe rephrase this without removal? Why is it better to not have this info at all?